engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-action-packages.controller.js

```diff
@@ -102,6 +107,8 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkActionPackag
                 HostBulkAction.updateContent(params, success, error);
             } else if ($scope.content.action === "remove") {
                 HostBulkAction.removeContent(params, success, error);
+            } else if ($scope.content.action === "update all") {
```

Comment:
We need to handle the case below for the remote execution provider (see `$scope.performViaRemoteExecution` below in this file).  Since it's just mapped to a yum command and the packages list should be blank, just transforming 'update all' back to 'update' might be enough.